docs/getting-started/12-bind-redis-service-to-function.md

```diff
@@ -0,0 +1,212 @@
+   content-length: 187
+   content-type: application/json; charset=utf-8
+   date: Mon, 13 Jul 2020 13:05:33 GMT
+   etag: W/"bb-Iuoc/aX9UjdqADJAZPNrwPdoraI"
+   server: istio-envoy
+   x-envoy-upstream-service-time: 25
+   x-powered-by: Express
+
+   [{"orderCode":"762727210","consignmentCode":"76272725","consignmentStatus":"PICKUP_COMPLETE"}, {"orderCode":"123456789","consignmentCode":"76272725","consignmentStatus":"PICKUP_COMPLETE"}]
+   ```
+
+   > **NOTE:** The listed orders are those created in the previous guides.
+
+3. Send a `POST` request to the Function with a sample order details:
+
+   ```bash
+   curl -ikX POST "https://$FUNCTION_DOMAIN" \
+     -H "Content-Type: application/json" \
+     -H 'Cache-Control: no-cache' -d \
+     '{
+       "orderCode": "762727234",
+       "consignmentCode": "76272725",
+       "consignmentStatus": "PICKUP_COMPLETE"
+     }'
+   ```
+
+4. When you call the `https://$FUNCTION_DOMAIN` URL again, the system returns a response similar to this one:
+
+   ```bash
+   HTTP/2 200
+   access-control-allow-origin: *
+   content-length: 280
+   content-type: application/json; charset=utf-8
+   date: Mon, 13 Jul 2020 13:05:51 GMT
+   etag: W/"118-lAc/HJqUaWFKlQ/uyvrhuPb++80"
+   server: istio-envoy
+   x-envoy-upstream-service-time: 9
+   x-powered-by: Express
+
+   [{"orderCode":"762727234","consignmentCode":"76272725","consignmentStatus":"PICKUP_COMPLETE"}, {"orderCode":"762727210","consignmentCode":"76272725","consignmentStatus":"PICKUP_COMPLETE"}, {"orderCode":"123456789","consignmentCode":"76272725","consignmentStatus":"PICKUP_COMPLETE"}]
+   ```
+
+   You can see the Function returns the previously sent order details.
+
+5. Remove the [Pod](https://kubernetes.io/docs/concepts/workloads/pods/) created by the `orders-function` Function. Run this command and wait for the system to delete the Pod and start a new one:
+
+   ```bash
+   kubectl delete pod -n orders-service -l "serverless.kyma-project.io/function-name=orders-function"
+   ```
+
+6. Call the Function again to check the storage:
+
+   ```bash
+   curl -ik "https://$FUNCTION_DOMAIN"
+   ```
+
+   The system returns a response similar to this one:
+
+   ```bash
+   HTTP/2 200
+   access-control-allow-origin: *
+   content-length: 280
+   content-type: application/json; charset=utf-8
+   date: Mon, 13 Jul 2020 13:05:33 GMT
+   etag: W/"118-lAc/HJqUaWFKlQ/uyvrhuPb++80"
+   server: istio-envoy
+   x-envoy-upstream-service-time: 975
+   x-powered-by: Express
+
+   [{"orderCode":"762727234","consignmentCode":"76272725","consignmentStatus":"PICKUP_COMPLETE"}, {"orderCode":"762727210","consignmentCode":"76272725","consignmentStatus":"PICKUP_COMPLETE"}, {"orderCode":"123456789","consignmentCode":"76272725","consignmentStatus":"PICKUP_COMPLETE"}]
+   ```
+
+   As you can see, `orders-function` uses Redis storage now. It means that every time you delete the Pod of the Function or change its Deployment definition, the order details will not get lost.
```

Comment:
```suggestion
   As you can see, `orders-function` uses Redis storage now. It means that every time you delete the Pod of the Function or change its Deployment definition, the order details are preserved.
```
See [this comment](https://github.com/kyma-project/kyma/pull/9296/files#r480121553) (not the suggestion, just how you phrased the line), and [this comment](https://github.com/kyma-project/kyma/pull/9296#discussion_r478412987) for the reason why I changed it.